testmain_test.go

```diff
@@ -69,11 +69,15 @@ func TestVerifyTestMain(t *testing.T) {
 
 	VerifyTestMain(dummyTestMain(7), RunOnFailure())
 	assert.Equal(t, 7, <-exitCode, "Exit code should not be modified")
-	assert.Contains(t, <-stderr, "goleak: Errors", "Find leaks on unsuccessful runs with RunOnFailure specified")
+	assert.Contains(t, <-stderr, "goleak: Errors on unsuccessful test run", "Find leaks on unsuccessful runs with RunOnFailure specified")
```

Comment:
`goleak: Errors` wasn't enough as it was hiding `RunOnFailure can only be passed to VerifyTestMain` in this case.